poms/bom-deployment/pom.xml

```diff
@@ -35,7 +35,6 @@
     <properties>
         <camel-quarkus.version>1.1.0-SNAPSHOT</camel-quarkus.version><!-- kept in sync with project.version by the release plugin -->
 
-        <jandex-maven-plugin.version>1.0.6</jandex-maven-plugin.version>
```

Comment:
this jandex thing seems unrelated to the pr, can we make this change in a separate commit ?

Reply:
Uneeded indeed. Let me remove it.